chia/wallet/dao_wallet/dao_wallet.py

```diff
@@ -321,12 +296,12 @@ async def create_new_dao_wallet_from_coin_spend(
         self.log = logging.getLogger(name if name else __name__)
 
         self.log.info(f"Creating DAO wallet from a coin spend {launch_coin.name()}  ...")
-        # Create did info from the coin spend
+        # Create dao info from the coin spend
```

Comment:
Do we want to comment out this whole function given we aren't using it or testing it and it probably doesn't work?